pkg/operator/ceohelpers/util.go

```diff
@@ -0,0 +1,46 @@
+package ceohelpers
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ghodss/yaml"
+
+	corev1listers "k8s.io/client-go/listers/core/v1"
+
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/operatorclient"
+)
+
+const (
+	clusterConfigName = "cluster-config-v1"
+	clusterConfigKey  = "install-config"
+)
+
+type replicaCountDecoder struct {
+	ControlPlane struct {
+		Replicas string `yaml:"replicas,omitempty"`
+	} `yaml:"controlPlane,omitempty"`
+}
+
+// GetMastersReplicaCount get number of expected masters statically defined by the controlPlane replicas in the install-config.
+func GetMastersReplicaCount(configmapLister corev1listers.ConfigMapLister) (int, error) {
```

Comment:
I also have this function in my PR. Could you open a new PR with just this function so that we can merge it before this PR? That would allow me to reuse it and would keep my PR smaller.

Reply:
I have created one https://github.com/openshift/cluster-etcd-operator/pull/767, ptal